### ISSUE TITLE
Hover for RadioCardGroup

### ISSUE THREAD

**Issue by anastasialanz** [dequelabs/cauldron#777]
When viewing Cauldron with the dark theme, in the [RadioCardGroup demo](https://cauldron.dequelabs.com/components/RadioCardGroup), the hover state color contrast for the middle card doesn't meet the color contrast minimum.

"No" text: `#ffffff`
Background color: `#e0e0e0`

Screenshot from Chrome Dev Tools:
<img width="963" alt="Screen Shot 2022-09-16 at 1 41 28 PM" src="https://user-images.githubusercontent.com/3081483/190717878-4add0589-3076-4e1a-90f2-1f29a30788f6.png">

https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html

**Comment by schne324** [comment]
@awpearlm what is the expected color here?

**Comment by schne324** [comment]
Important to note that this is just the :hover state for **disabled** radio card group items (I've updated the title accordingly)

**Comment by anastasialanz** [comment]
Why does only the disabled one have a hover state?

**Comment by anastasialanz** [comment]
@scurker Can we close this since RadioCardGroup is deprecated now?

**Comment by scurker** [comment]
It is a valid accessibility issue that is still present so I don't think we want to lose site of it until the pattern has been removed and we have a replacement pattern. It's still on our todo list to do both.

**Comment by anastasialanz** [comment]
@scurker I linked the new replacement component in the description incase this doesn't get fixed before the replacement happens.
